modin/config/envvars.py

```diff
@@ -170,7 +170,7 @@ class GpuCount(EnvironmentVariable, type=int):
 
 
 class Memory(EnvironmentVariable, type=int):
-    """How much memory give to each Ray worker (in bytes)."""
+    """How much memory give to each Ray/Dask worker (in bytes)."""
```

Comment:
```suggestion
    """How much memory give to each Ray/Dask worker (in bytes) depending on used CPUs."""
```
Should probably be clarified.

Reply:
In Ray's case, we don't depend on CPUs, are we?

Reply:
Yes, you are correct. The docstring should be something like this then:

```
"""
How much memory (in bytes) give to an execution engine.

Notes
-----
* In the case of Ray execution engine the provided memory is used to start the plasma object store.
* In the case of Dask execution engine the provided memory is given to each worker depending on CPUs used.
"""
```
Corrections are welcome.

Reply:
Docstring is updated.